app/services/record_declaration.rb

```diff
@@ -156,10 +156,23 @@ def declaration_parameters
       declaration_type:,
       cpd_lead_provider:,
       delivery_partner:,
+      mentor_user_id: mentor&.id,
       user: participant_identity.user,
     }
   end
 
+  def mentor
+    return nil unless participant_profile.ect?
+
+    induction_record = Induction::FindBy.call(
+      participant_profile:,
+      lead_provider: cpd_lead_provider.lead_provider,
+      date_range: (Date.new(0)..declaration_date),
```

Comment:
not that it matters here but would `..declaration_date` make sense here instead of the zeroed date?